templates/layout/includes/microsites-footer.html.twig

```diff
@@ -80,42 +38,42 @@
   <div class="lgd-region__inner lgd-region__inner--{{ region|clean_class }}">
     {# Begin Footer text blocks #}
     <div class="lgd-row">
-      {% if microsite_footer_text_block_1 %}
+      {% if microsites.footer.items.lgms_footer_text_block_1 %}
```

Comment:
text blocks, footer banner and footer logos don't appear to be printing either :(